### PR TITLE
GitHub actions salesforce automation

### PR DESCRIPTION
**Pull request by pradeepchd**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Copyright © 2021 Cask Data, Inc.
```
Comment (rmstar) [inline .github/workflows/e2e.yml]: 2022

File: .github/workflows/trigger.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Copyright © 2021 Cask Data, Inc.
```
Comment (rmstar) [inline .github/workflows/trigger.yml]: 2022

**Review by rmstar — APPROVED** [review]
please squash commits after addressing minor comment

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (poojantcs) [reply .github/workflows/e2e.yml]: Updated.

File: .github/workflows/trigger.yml (hunk 2 above)
Comment (poojantcs) [reply .github/workflows/trigger.yml]: Updated.

**Comment by poojantcs** [comment]
> please squash commits after addressing minor comment

Hi @rmstar , I have addressed the comments and squashed the commits.

**Review by rmstar — APPROVED** [review]

**Pull request merged**